### PR TITLE
Fix breaking GIFs while removing metadata

### LINKED ISSUE #4109: Don't break GIFs with metadata removal
When uploading files thelounge checks whether it is an image type file and if so it renders the image in a canvas to get rid of all the metadata. This is all nice and well but breaks GIFs as it only renders the first frame. 

https://github.com/thelounge/thelounge/blob/5a1963647e9ee095a0b0bc585bf23f11169ca48a/client/js/upload.js#L141-L149

I propose to add a second exception for GIFs as for SVGs. GIFs do not support any EXIF data nor do they have any other kind of metadata that isn't important for the GIF to function properly.

### PR DESCRIPTION
**Pull request by Nachtalb**
Closes #4109
GIFs can't contain EXIF data and do not contain any other metadata that isn't operational

### CODE REVIEW

**Review by MaxLeiter — APPROVED** [review]

File: client/js/upload.js [hunk 1]
```diff
@@ -141,7 +141,8 @@ class Uploader {
 		if (
 			store.state.settings.uploadCanvas &&
 			file.type.startsWith("image/") &&
-			!file.type.includes("svg")
+			!file.type.includes("svg") &&
+			!file.type.includes("gif")
```
Comment (xPaw) [inline client/js/upload.js:145]: Can't this just be `file.type === 'image/gif'`?

Comment (Nachtalb) [reply client/js/upload.js:145]: I just went with the way it has been done for SVGs 🤷 , either way would work. Should I change it?

Comment (xPaw) [reply client/js/upload.js:145]: Sure. The svg uses `includes` because the mime type is incosistent as far as I remember.

Comment (Nachtalb) [reply client/js/upload.js:145]: Not according to https://www.iana.org/assignments/media-types/media-types.xhtml, shall I change both?

Comment (xPaw) [reply client/js/upload.js:145]: I remember there was something specific to svg that had to be this way, I can't remember now though, so just keep it.

**Review by xPaw — APPROVED** [review]

**Pull request merged**